build_info/img4tool.control

```diff
@@ -2,7 +2,7 @@ Package: img4tool
 Maintainer: @DEB_MAINTAINER@
 Architecture: @DEB_ARCH@
 Version: @DEB_IMG4TOOL_V@
-Depends: libplist, openssl, libgeneral
+Depends: libplist, openssl, libgeneral, libimg4tool0
```

Comment:
Could you have this depend on libssl1.1 and libplist3 rather than what it currently does?